test/verify/machineslib.py

```diff
@@ -1166,23 +1166,24 @@ class TestCreateConfig:
 
     class VmDialog:
 
-        def __init__(self, test_obj, name, is_filesystem_location=True, location='',
+        def __init__(self, test_obj, name, sourceType='file', location='',
                      memory_size=1, memory_size_unit='GiB',
                      storage_size=1, storage_size_unit='GiB',
                      os_vendor=None,
                      os_name=None,
                      start_vm=False,
                      connection=None):
 
-            if not is_filesystem_location and start_vm:
+            if sourceType == 'url' and start_vm:
                 raise Exception("cannot start vm because url specified (no connection available in this test)")
 
             self.browser = test_obj.browser
             self.machine = test_obj.machine
             self.assertTrue = test_obj.assertTrue
 
             self.name = name
-            self.is_filesystem_location = is_filesystem_location
+            self.is_filesystem_location = sourceType == 'file'
```

Comment:
Is this still actually needed then? Seems better to consistently change all tests to use `sourceType`?

Reply:
done